inst/extdata/motrpac_colors_abbr.R

```diff
@@ -1,17 +1,24 @@
 
 # Motrpac colors abbreviations
 
-assay_abbr = c(atacseq='ATAC',
+assay_abbr = c(atac='ATAC',
+               atacseq='ATAC',
                'atac-seq'='ATAC',
+               
```

Comment:
Why these spaces between the different key/values? Would it be possible to remove them (all over the file ;-)?
It would be great to keep a nice style (code style parsers might complain)

Reply:
done!

Reply:
Nice, thanks!